app/src/main/java/org/simple/clinic/search/PatientSearchUpdate.kt

```diff
@@ -1,14 +1,33 @@
 package org.simple.clinic.search
 
 import com.spotify.mobius.Next
+import com.spotify.mobius.Next.noChange
 import com.spotify.mobius.Update
 import org.simple.clinic.mobius.next
 
 class PatientSearchUpdate : Update<PatientSearchModel, PatientSearchEvent, PatientSearchEffect> {
 
   override fun update(model: PatientSearchModel, event: PatientSearchEvent): Next<PatientSearchModel, PatientSearchEffect> {
-    return when(event) {
+    return when (event) {
       is SearchQueryTextChanged -> next(model.queryChanged(event.text))
+      is SearchClicked -> {
+        val validationErrors = validateInput(model.enteredQuery)
```

Comment:
Can we extract this in a private function?

Reply:
It was done in a later commit in the same PR.